Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:0.12.5
+RUN apt-get update && apt-get install -y --no-install-recommends python3-minimal python3-virtualenv python3-pip python3-dev python3-pil
+RUN python3 -m virtualenv -p /usr/bin/python3 --system-site-packages /var/www/shoop/venv
+RUN /var/www/shoop/venv/bin/pip install -U pip
+ADD . /var/www/shoop/working_copy
+RUN /var/www/shoop/venv/bin/pip install /var/www/shoop/working_copy
+RUN SHOOP_WORKBENCH_DISABLE_MIGRATIONS=1 /var/www/shoop/venv/bin/python -m shoop_workbench migrate
```

Comment:
What would be the workflow of someone developing using the image anyway? Besides, they could just use `migrate --fake` to fix the migration history table; using `DISABLE_MIGRATIONS` just makes Django use the faster logic that Just Creates Tables.